packages/vuetify/src/mixins/intersectable/index.ts

```diff
@@ -0,0 +1,43 @@
+import Vue from 'vue'
+import { consoleWarn } from '../../util/console'
+
+export default function intersectable (options: { onVisible: string[] }) {
+  if (!('IntersectionObserver' in window)) {
```

Comment:
this will break on SSR

Reply:
Because of the `window`?

Reply:
You have to check for the existence of the window